internal/git/git.go

```diff
@@ -51,6 +52,16 @@ func Run(ctx context.Context, args ...string) (string, error) {
 	return RunWithEnv(ctx, []string{}, args...)
 }
 
+func RunCmds(ctx context.Context, cwd string, env []string, cmds [][]string) error {
```

Comment:
I don't think we need to move this here, it can probably live closer to where it's used.